browser/main/lib/MenuManager.js

```diff
@@ -0,0 +1,35 @@
+const OSX = global.process.platform === 'darwin'
```

Comment:
The reason why I added `MenuManager` instead of `ShortcutManager` is that because of a preparation of Plugin feature. I plan to be able to add to the menu.